File: Unidad 2/Meta 2.2/Java/Taller221.java
public class Taller221 {
    public static void main(String[] args) {
        Biosenosis biosenosis = new Biosenosis("Biosenosis");
        Biotopo biotopo = new Biotopo("Biotopo");
        SistemaBiologico sistemaBiologico = new SistemaBiologico("SistemaBiologico");
        Ecosistema ecosistema = new Ecosistema("Ecosistema");
        System.out.println(biosenosis);
        System.out.println(biotopo);
        System.out.println(sistemaBiologico);
        System.out.println(ecosistema);

        
        ecosistema.getBiotopos().add(biotopo);
        ecosistema.getBiosenosis().add(biosenosis);
        biotopo.setEcosistema(ecosistema);
        biosenosis.setEcosistema(ecosistema);

        System.out.println(ecosistema);

    }
    
}

File: Unidad 2/Meta 2.2/Python/Taller221.py
import Ecosistema as Ecosistema
import Biosenosis as Biosenosis
import Biotopo as Biotopo
import SistemaBiologico as SistemaBiologico

if __name__ == '__main__':
    biosenosis = Biosenosis.Biosenosis("Biosenosis 1")
    biotopo = Biotopo.Biotopo("Biotopo 1")
    sistemaBiologico = SistemaBiologico.SistemaBiologico("Sistema Biologico 1")
    ecosistema = Ecosistema.Ecosistema("Ecosistema 1")
    print(biosenosis)
    print(biotopo)
    print(sistemaBiologico)
    print(ecosistema)

    ecosistema.getBiosenosis().append(biosenosis)
    ecosistema.getBiotopos().append(biotopo)
    print(ecosistema)

    biosenosis2 = Biosenosis.Biosenosis("Biosenosis 2")
    biotopo2 = Biotopo.Biotopo("Biotopo 2")
    ecosistema.getBiosenosis().append(biosenosis2)
    ecosistema.getBiotopos().append(biotopo2)
    print(ecosistema)